src/main/java/core/basesyntax/dao/CarJdbcDaoImpl.java

```diff
@@ -99,7 +99,7 @@ public Car create(Car car) {
     public Car update(Car car) {
         String updateQuery = "UPDATE cars SET model = ?, manufacturer_id = ? "
                 + "WHERE car_id = ? AND deleted = FALSE";
-        String deleteQuery = "DELETE FROM cars_drivers WHERE car_id = ?";
+        String deleteQuery = "DELETE FROM cars_drivers WHERE car_id = ? AND driver_id IS NOT NULL";
```

Comment:
this should be handle by constraint in the table for this field

Reply:
I handled this in the table, but sometimes when I add a driver to car exception with such trace `cannot add or update a child row a foreign key constraint fails`  occurs
Anyway, I delete this